docs/rst/tutorials/dynamic_types.rst

```diff
@@ -198,39 +197,14 @@ First, it creates a ``TypeSupport`` with the corresponding type and registers it
 Then, it creates the DDS Topic with the topic name set in the creation of the Subscriber and the topic type previously registered.
 Finally, it creates the DataReader of that topic.
 
-.. literalinclude:: ../../../../resources/dds/TypeLookupService/TypeLookupServiceSubscriber.cpp
+.. literalinclude:: ../../../resources/dds/TypeLookupService/TypeLookupServiceSubscriber.cpp
     :language: C++
     :lines: 278-304
 
 ***********************
 Running the application
 ***********************
 
-Recording samples with DDS Recorder
-===================================
-
-Open two terminals:
-
-* In the first terminal, run an instance of the described publisher:
-
-    .. code-block:: bash
-
-        source install/setup.bash
-        cd DDS-Record-Replay/build/TypeLookupService
-        ./TypeLookupService
-
-* In the second terminal, run the ddsrecorder:
-
-    .. code-block:: bash
-
-        source install/setup.bash
-        ddsrecorder
-
-Stop the |ddsrecorder| at any time to save the output MCAP file containing the recorded data.
-
-Publisher <-> Subscriber
-========================
-
```

Comment:
Because now it is a generic tutorial for using dynamic types in Fast-DDS. How to launch a recorder is already shown in recorder's getting started, and in Foxglove tutorial.